src/group/managed_group/mod.rs

```diff
@@ -105,16 +141,58 @@ impl ManagedGroup {
         credential_bundle: &CredentialBundle,
         members: &[usize],
```

Comment:
I still don't like this API, having to pass in a member index. But let's tackle this after this PR got merged.